packages/cadl-ranch-specs/http/server/versions/non-operations/main.tsp

```diff
@@ -0,0 +1,22 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+
+using TypeSpec.Http;
+
+@doc("Illustrates server with non operations defined with api-versions.")
+@scenarioService("/server/versions/nonOperations")
+@server(
+  "{endpoint}",
+  "Testserver endpoint",
+  {
+    @doc("Need to be set as 'http://localhost:3000' in client.")
+    endpoint: url,
+  }
+)
+namespace Server.Versions.NonOperations;
+
+@scenario
```

Comment:
For the not versioned and every operation versioned as the scenario is basically the whole service can you move the `@secenario` and `@Sscenario Doc` decorator directly on the namespace instead of the operation. That way it won't have this extra nesting in the dashboard